docs/389ds/design/dna-plugin-proposal.md

```diff
@@ -33,37 +33,37 @@ Mighty ASCII art with 4 servers and only SRV 1 creating users, each server has o
 
-Last but not least this plugin requires all masters to be up and talking to make any configuration changes or possible conflict can easily come up if only some of the masters get the configuration update (think of network split case).
+Last but not least this plugin requires all suppliers to be up and talking to make any configuration changes or possible conflict can easily come up if only some of the suppliers get the configuration update (think of network split case).
 
 ### Requirements
 
-It must be possible to spread ID allocation between servers in a way that makes reconfiguring allocation ranges more easy. It must be possible to reconfigure masters even if not all of them are online at the same time.
+It must be possible to spread ID allocation between servers in a way that makes reconfiguring allocation ranges more easy. It must be possible to reconfigure suppliers even if not all of them are online at the same time.
 
 ### Possible Solutions
 
 It is certainly important to retain the ability to allocate IDs from a range or slice so that communication with other DCs is kept to a minimum and necessary only when ranges are depleted and reconfiguration is needed.
 
 Using ranges of IDs instead of an algorithm to split a uniquely available space would make this easier.
 
-Manually splitting the ID space into ranges would be easy. And assigning a range to each server too. The advantage of using a range is that if one server is running low on IDs all is needed is only one other master available that has a big enough range assigned. The 2 servers can in fact be reconfigured so that half of the range previously assigned to one server can be reassigned to the other without the need for other masters to be online. The change of ownership information will be replicated to them as soon as they come up. There is no possibility that one master will steal ranges from another one as communication between the new and the old owner is required to allow the transfer of ownership.
+Manually splitting the ID space into ranges would be easy. And assigning a range to each server too. The advantage of using a range is that if one server is running low on IDs all is needed is only one other supplier available that has a big enough range assigned. The 2 servers can in fact be reconfigured so that half of the range previously assigned to one server can be reassigned to the other without the need for other suppliers to be online. The change of ownership information will be replicated to them as soon as they come up. There is no possibility that one supplier will steal ranges from another one as communication between the new and the old owner is required to allow the transfer of ownership.
```

Comment:
I have a doubt here with the terminology of the DNA plugin. With DNA a server can request range to another server. I do not know how it was express in DNA plugin description. We should stick to what is in the doc

Reply:
I checked both Admin and Config guides and we use the supplier terminology there.